### ISSUE TITLE
[no-bare-strings] Detection of constant string in page-title helper

### ISSUE THREAD

**Issue by mhankus** [ember-template-lint/ember-template-lint#1886]
Currently no-bare-strings does not check helpers. It would be great, if it could detect use of constant strings in selected helper arguments. For example page-title helper is currently used in ember blueprints, but is not verified by linter if it was internationalized 

Sample code which should be detected
```
{{page-title "Foo"}}
````

**Comment by lifeart** [comment]
likely we should skip paths "foo.bar" for this check (if will be added)

**Comment by bertdeblock** [comment]
@lifeart Do you mean e.g. using the `t` helper from `ember-intl` like so: `{{page-title (t "foo.bar")}}`?

@rwjblue Do we want to extend the `no-bare-strings` rule for this case? If so, I can pick this up.

**Comment by runspired** [comment]
I think there's a more general issue here, I'm observing developers do the following to avoid triggering this rule:

```hbs
<h2>{{"Contact"}}</h2>
```

**Comment by bertdeblock** [comment]
@runspired Normally, this should already be covered:
https://github.com/ember-template-lint/ember-template-lint/blob/master/test/unit/rules/no-bare-strings-test.js#L125-L133

**Comment by lifeart** [comment]
Closing as this cases already covered and relate PR is merged.
